### DISCUSSION TITLE
How to make createOptionForm nested?

### DISCUSSION THREAD

**Discussion by guuhgalvao** [filamentphp/filament#8827] [Help]
Hey devs and legends,

I need you. I'm trying to make a createOptionForm within another, but this makes the modal disappear.

Has anyone been through this and could help me?

**Comment by ArtMin96** [comment]
You can try something similar. I don't even know if this works or not, I'm just giving an idea.
[https://filamentphp.com/docs/3.x/actions/modals#opening-another-modal-from-an-extra-footer-action](https://filamentphp.com/docs/3.x/actions/modals#opening-another-modal-from-an-extra-footer-action)

```php
Select::make('column')
    ->suffixAction(
        Action::make('nestedModal')
            ->icon('heroicons-o-plus')
            ->form([
                TextInput::make...
            ])
            ->action(),
    ),
```

**Reply by guuhgalvao** [reply]
Unfortunately, it doesn't work. I'm using version 2. But, thanks for the sugestion!

**Reply by ArtMin96** [reply]
you can't nest modals in v2

**Reply by guuhgalvao** [reply]
Seriously? How sad :(
